tests/pipeline/test_technical.py

```diff
@@ -138,3 +141,36 @@ def test_bollinger_bands_output_ordering(self):
         self.assertIs(lower, bbands.lower)
         self.assertIs(middle, bbands.middle)
         self.assertIs(upper, bbands.upper)
+
+
+class AroonTestCase(ZiplineTestCase):
+    window_length = 10
+    nassets = 5
+    dtype = [('down', 'f8'), ('up', 'f8')]
+
+    @parameterized.expand([
+        (np.arange(window_length),
+         np.arange(window_length) + 1,
+         np.recarray(shape=(nassets,), dtype=dtype,
+                     buf=np.array([0, 100] * nassets, dtype='f8'))),
+        (np.arange(window_length, 0, -1),
+         np.arange(window_length, 0, -1) - 1,
+         np.recarray(shape=(nassets,), dtype=dtype,
+                     buf=np.array([100, 0] * nassets, dtype='f8'))),
+        (np.array([10, 10, 10, 1, 10, 10, 10, 10, 10, 10]),
+         np.array([1, 1, 1, 1, 1, 10, 1, 1, 1, 1]),
+         np.recarray(shape=(nassets,), dtype=dtype,
+                     buf=np.array([100 * 3 / 9,100 * 5 / 9] * nassets,
+                                  dtype='f8'))),
+    ])
+    def test_aroon_basic(self, lows, highs, expected_out):
+        aroon = Aroon(window_length=self.window_length)
+        today = pd.Timestamp('2014')
```

Comment:
While it may not matter, we will probably be passing a utc timestamp like: `Timestamp('2014', tz='utc')`